sat/cli/bootsys/cabinet_power.py

```diff
@@ -201,6 +206,13 @@ def do_cabinets_power_on(args):
                      f'rescheduled after resume: {err}')
         raise SystemExit(1)
 
+    LOGGER.info('Recreating cronjobs which have failed to have been scheduled on time.')
+    try:
+        batch_api = load_kube_api(api_cls=BatchV1Api)
+        recreate_namespaced_stuck_cronjobs(batch_api, 'services')
+    except ConfigException as err:
+        LOGGER.warning('Could not load Kubernetes configuration: %s', err)
+
```

Comment:
Now that you are re-creating a possibly stuck hms-discovery cronjob in the `HMSDiscoveryScheduledWaiter` in the `on_retry_action`, you wouldn't want to unconditionally re-create the cronjob here, would you?